app/src/cc/arduino/contributions/libraries/ui/ContributedLibraryTableCellJPanel.java

```diff
@@ -196,7 +196,7 @@ public ContributedLibraryTableCellJPanel(JTable parentTable, Object value,
         desc += format("{0}", paragraph);
       desc += "<br />";
     }
-    if (author != null && !author.isEmpty()) {
+    if (maintainer != null && !maintainer.isEmpty()) {
       desc = setButtonOrLink(moreInfoButton, desc, moreInfoLbl, website);
     }
```

Comment:
Not strictly related to this change,  but why does this if exist? Seems like you don't get a "More info" button/link when no author (or now maintainer) is set, but that doesn't make sense to me?

Reply:
You're right, it should be `website != null && !website.isEmpty()`, I guess we may fix it together with this PR.